infrastructure/docker-compose.yml

```
@@ -40,4 +40,59 @@ services:
+    container_name: genieus-kafka-ui
+    image: provectuslabs/kafka-ui:latest
+    ports:
+      - "8085:8085"
+    environment:
+      SERVER_PORT: 8085
+      KAFKA_CLUSTERS_0_NAME: Genieus-Local-Cluster
+      KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: kafka-1:9092,kafka-2:9092
+    depends_on:
+      - zookeeper
+      - kafka-1
+      - kafka-2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)